docs/videos/live-app-review.md

```diff
@@ -213,6 +254,13 @@ sidebar_label: Application Reviews
   allowfullscreen>
 </iframe>
 
+> **Dropping tokens to users in the traditional Web2 way!**
+>
+> - One of the issues with Social Tokens is that they start with zero value. A creator, artist or community might want to drop a bunch of tokens to their fans but the audience has (1) no crypto to pay for fees (2) no wallet (3) no concept of crypto or blockchain; prior to the drop.
+> - We do a demo of creating a "guest" named account for an app where the gas fees are sponsored by a special app account called "guests.APP_NAME.near".
+> - The guest account doesn't exist (sometimes called a virtual or contract account) until the user decides to swap their tokens and upgrade to a real account. Until then their name is reserved because only the app is able to create "USERNAME.APP_NAME.near".
```

Comment:
```suggestion
> - The guest account doesn't exist (sometimes called a virtual or contract account) until the user decides to swap their tokens and upgrade to a real account. Until then their name is reserved because only the app is able to create `USERNAME.APP_NAME.near`.
```